### DISCUSSION TITLE
How can I use new bing in China

### DISCUSSION THREAD

**Discussion by Wczzd** [community/community#106762] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I don't know so I ask you.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Wczzd ! We're happy you're here. The Discussions category is solely related to conversations around the GitHub product Discussions.


If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Bing . It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
